.gitignore

```diff
@@ -8,4 +8,5 @@ shippable/
 .tox/
 .coverage*
 .mypy_cache/
-
+# htmlcov is generated by tox due to coverage gathering in pytest
+htmlcov/
```

Comment:
as this is generated as part of the build, could you explain the reason for a dedicated folder in the root of the repo, instead of having this content in the build folder ?

Reply:
This is just the default directory created by the coverage test engine.